docs/configdoc.md

```diff
@@ -0,0 +1,104 @@
+|metrics-addr|`"0.0.0.0:8898"`|Prometheus endpoint for pull/scrape the metrics, for more info about Prometheus check [prometheus.io](https://prometheus.io/) | Yes |
+|kf-poll-interval|`"30s"`|Periodic Interval to scrape metrics using Prometheus| Yes |
+|n-metric-samples|`"20"`|Number of Metric Samples| Yes |
+
+
+## [xdp-root-program]
+| FieldName     | Example       | Description     | Required        |
+| ------------- | ------------- | --------------- | --------------- |
+|name|`"xdp-root"`|Name of xdp-root program| Yes |
+|artifact|`"xdp-root.tar.gz"`|Archive of xdp-root program. Only tar.gz and .zip formats are supported| Yes |
+|ingress-map-name|`"root_array"`|Ingress map name of xdp-root program| Yes |
+|command|`"xdp_root"`|Command to run xdp-root program| Yes |
+|version|`"1.01"`|Version of xdp-root program| Yes |
+|user-program-daemon|`"false"`|Boolean to check xdp-root is user-program daemon or not| Yes |
+
+## [tc-root-program]
+| FieldName     | Example       | Description     | Required        |
+| ------------- | ------------- | --------------- | --------------- |
+|name|`"tc_root"`|Name of tc_root program| Yes |
+|artifact|`"l3af_tc_root.tar.gz"`|Archive of tc_root program| Yes |
+|ingress-map-name|`"tc_ingress_root_array"`|Ingress map name of tc_root program| Yes |
+|egress-map-name|`"tc_egress_root_array"`|Egress map name of tc_root program,for more info about ingress/egress check [cilium](https://docs.cilium.io/en/v1.9/concepts/ebpf/intro/)| Yes |
+|command|`"tc_root"`|Command to run tc_root program| Yes |
+|version|`"1.0"`|Version of tc_root program| Yes |
+|user-program-daemon|`"false"`|Boolean to check xdp-root is user-program daemon or not| Yes |
+
+## [l3af-configs]
+| FieldName     | Example       | Description     | Required        |
+| ------------- | ------------- | --------------- | --------------- |
+|restapi-addr|`"localhost:53000"`| Hostname and Port of l3af-configs rest api | Yes |
+
+# [l3af-config-store]
+| FieldName     | Example       | Description     | Required        |
+| ------------- | ------------- | --------------- | --------------- |
+|filename|`"/etc/l3afd/l3af-config.json"`|Absolute path of persistent config file where we are storing L3afBPFPrograms object's , for more info check [models](https://github.com/l3af-project/l3afd/blob/main/models/l3afd.go)| Yes |
+
+# [mtls]
+| FieldName     | Example       | Description     | Required        |
+| ------------- | ------------- | --------------- | --------------- |
+|enabled| `"true"` | Boolean to check mtls enabled or not on REST API exposed by l3afd| Yes |
+|min-tls-version|`"1.3"`| Minimum tls version allowed| No |
+|cert-dir|`"/etc/l3af/certs"`|Absolute path of ca certificates. In Windows we store ca certificates in [certificate store](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/certificate-stores),but in l3afd context we will store in filesystem.This is the limitation of l3afd.  | Yes |
```

Comment:
```suggestion
|cert-dir|`"/etc/l3af/certs"`|Absolute path of ca certificates. In Windows we store ca certificates in [certificate store](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/certificate-stores),but in l3afd context we will store in filesystem.This is the limitation of l3afd.  | Yes |
```
This text seems contradictory.  It implies that on windows l3afd stores certs in the certificate store, but in a l3afd windows context, they're not.   That sounds like a contradiction.  Please reword.